build/development-guide/smart-contracts/get-started-evm/use-hardhat.md

```diff
@@ -0,0 +1,96 @@
+# Using Hardhat to deploy on Acala, Karura & Mandala
+
+This example is inpired by the [Getting Started](https://hardhat.org/getting-started/) section of Hardhat Documentation.  
+
+## Prerequisites
+- node
+- yarn
+- npx
+
+
+## Let's create a basic sample project
+
+```bash=
+mkdir hardhat && cd hardhat
+yarn add hardhat
+npx hardhat
+```
+
+```bash=
+888    888                      888 888               888
+888    888                      888 888               888
+888    888                      888 888               888
+8888888888  8888b.  888d888 .d88888 88888b.   8888b.  888888
+888    888     "88b 888P"  d88" 888 888 "88b     "88b 888
+888    888 .d888888 888    888  888 888  888 .d888888 888
+888    888 888  888 888    Y88b 888 888  888 888  888 Y88b.
+888    888 "Y888888 888     "Y88888 888  888 "Y888888  "Y888
+
+Welcome to Hardhat v2.6.8
+
+✔ What do you want to do? · Create a basic sample project
+✔ Hardhat project root: · /home/.../hardhat
+✔ Do you want to add a .gitignore? (Y/n) · y
+✔ Do you want to install this sample project's dependencies with yarn (@nomiclabs/hardhat-waffle ethereum-waffle chai @nomiclabs/hardhat-ethers ethers)? (Y/n) · y
+```
+
+## Add the RPC nodes
+
+Add the networks section to the `hardhat.config.js` file inside the `module.exports` like so:
+
+```javascript=
+module.exports = {
+  solidity: "0.8.4",
+  networks: {
+    development: {
+      url: 'http://localhost:8545',
+      chainId: 595,
+      // Development built-in default deployment account
+      accounts: ["0xa872f6cbd25a0e04a08b1e21098017a9e6194d101d75e13111f71410c59cd57f"]
+    }
```

Comment:
we need to specify a gas price so the contract can deploy, otherwise there will be a `transaction is outdated` error. Like here https://github.com/AcalaNetwork/hardhat-tutorials/blob/master/hello-world/hardhat.config.js#L16